c++/test/TestPredicateLeaf.cc

```diff
@@ -145,6 +145,22 @@ namespace orc {
     return colStats;
   }
 
+  static proto::ColumnStatistics createTimestampStats(
+                                    int64_t minSecond, int32_t minNano,
```

Comment:
Just a question. Do we have an indentation rule in Apache ORC C++ module?

Reply:
I think we have: https://orc.apache.org/develop/coding/